### ISSUE TITLE
Leader/Follower Questions

### ISSUE THREAD

**Issue by Bchilled** [blizzhackers/kolbot#66]
My goal is to play my hammerdin, and have a wind druid follow me anywhere i go and fight along side me and pickup stuff. 

I cant seem to figure this out.

Right now i have the leader set to follow and basicly bot while i research and look up how to make my main goal work, but im having some issues.

my druid takes the tp my paladin gives, and then casts all his bo's pets hearts and basicly miss's all the action and is no help

also both of them just sell everything even thow i setup a pickit list and configured inventory.

to sum up my questions

1. how can i make my hammerdin maunaly played with my druid support (bot)

2. On a serpate config how do i get my druid (follower) to pre buff himself while waiting for the portals.

**Comment by mf022** [comment]
https://github.com/blizzhackers/documentation/blob/master/kolbot/MultiBotting.md/#using-followers
using the default [kolton's Follower.js](https://github.com/blizzhackers/kolbot/blob/master/d2bs/kolbot/libs/bots/Follower.js), you should type commands.

> and then casts all his bo's pets heart
for windy druid, only Oak sage is enough.
cyclone armor, and hurricane are rebuffed during druid attack.
